### DISCUSSION TITLE
[Un]staging hunks or lines from the preview?

### DISCUSSION THREAD

**Discussion by bennypowers** [tanvirtin/vgit.nvim#208] [Q&A] [answered]
Hello! I'm a refugee from Atom, and in my few weeks of using neovim as my daily driver, VGit comes the closest to replacing Atom's excellent GitHub plugin as an easy to use visual git client.

One thing I'm missing is the ability to stage or unstage hunks from the preview window:

```vim
:VGit project_hunks_preview
<C-j>
" somehow stage or unstage this hunk
```

Perhaps I'm misinterpreting the intention here. Any guidance you can offer is appreciated.

**Comment by tanvirtin** [comment, marked as answer]
Hi, thanks for the feedback. Glad to hear you are finding this plugin useful as you are transitioning to Neovim. Currently there's an issue open: https://github.com/tanvirtin/vgit.nvim/issues/170 which is addressing the same.

It's one of the features that is top of my priority list to work on.
